app/models/activity.rb

```diff
@@ -29,7 +29,6 @@ def content_type
 
   # Disabled validations
   # validates :end_date
-  # validates :description
```

Comment:
Why should this comment be removed?

Reply:
Forgot to replace it there, but I also think this comment section is no longer needed since it has not been touched in the last 3 years

Reply:
In that case just remove the whole block